IndexerGUI/Properties/Resources.Designer

```diff
@@ -0,0 +1,80 @@
+    // with the /str option, or rebuild your VS project.
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "15.0.0.0")]
+    [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
+    [global::System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
+    internal class Resources {
+        
+        private static global::System.Resources.ResourceManager resourceMan;
+        
+        private static global::System.Globalization.CultureInfo resourceCulture;
+        
+        [global::System.Diagnostics.CodeAnalysis.SuppressMessageAttribute("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
+        internal Resources() {
+        }
+        
+        /// <summary>
+        ///   Returns the cached ResourceManager instance used by this class.
+        /// </summary>
+        [global::System.ComponentModel.EditorBrowsableAttribute(global::System.ComponentModel.EditorBrowsableState.Advanced)]
+        internal static global::System.Resources.ResourceManager ResourceManager {
+            get {
+                if (object.ReferenceEquals(resourceMan, null)) {
+                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Indexer.Properties.Resources", typeof(Resources).Assembly);
+                    resourceMan = temp;
+                }
+                return resourceMan;
+            }
+        }
+        
+        /// <summary>
+        ///   Overrides the current thread's CurrentUICulture property for all
+        ///   resource lookups using this strongly typed resource class.
+        /// </summary>
+        [global::System.ComponentModel.EditorBrowsableAttribute(global::System.ComponentModel.EditorBrowsableState.Advanced)]
+        internal static global::System.Globalization.CultureInfo Culture {
+            get {
+                return resourceCulture;
+            }
+            set {
+                resourceCulture = value;
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to &lt;UserSettings xmlns=&quot;http://schemas.datacontract.org/2004/07/Indexer&quot; xmlns:i=&quot;http://www.w3.org/2001/XMLSchema-instance&quot;&gt;
+        ///  &lt;ExcludeFiles&gt;false&lt;/ExcludeFiles&gt;
+        ///  &lt;ExcludeFolders&gt;false&lt;/ExcludeFolders&gt;
+        ///  &lt;ExcludeHiddenAndSystem&gt;true&lt;/ExcludeHiddenAndSystem&gt;
+        ///  &lt;FiltersVisibility&gt;Collapsed&lt;/FiltersVisibility&gt;
+        ///  &lt;IsFiltersExpanded&gt;false&lt;/IsFiltersExpanded&gt;
+        ///  &lt;SelectedDrives xmlns:a=&quot;http://schemas.microsoft.com/2003/10/Serialization/Arrays&quot;&gt;&lt;/SelectedDrives&gt;
+        ///  &lt;WndHeight&gt;700&lt;/WndHeight&gt;
+        ///  &lt;WndWidth&gt;1000 [rest of string was truncated]&quot;;.
+        /// </summary>
+        internal static string defaultUserSettings {
```

Comment:
In the designer, it is still lower case. Better to recreate the resource from scratch.